src/analysis/mod.rs

```diff
@@ -39,6 +40,7 @@ pub struct Analysis {
     pub objects: BTreeMap<String, object::Info>,
     pub records: BTreeMap<String, record::Info>,
     pub global_functions: Option<info_base::InfoBase>,
+    pub constants: Option<info_base::InfoBase>,
```

Comment:
Maybe better just use `Vec<constants::Info>` and remove `constants` from `InfoBase` ?

Reply:
makes sense